services/actions/notifier_helper.go

```diff
@@ -144,6 +145,17 @@ func notify(ctx context.Context, input *notifyInput) error {
 		return fmt.Errorf("gitRepo.GetCommit: %w", err)
 	}
 
+	if input.Event == webhook_module.HookEventPush || input.Event == webhook_module.HookEventPullRequest {
```

Comment:
In Gitea, when a new commit is pushed to PR's head branch, the webhook event is called `HookEventPullRequestSync`, which should also trigger this `skip ci` functionality. See Github example in my test repo(https://github.com/lng2020/fork_repo/pull/1)

Reply:
Oh, seems that I misunderstood their docs. I added pull_request_sync